client/src/syncrepl.rs

```diff
@@ -133,9 +133,11 @@ impl LdapClient {
                     // state of the content after performing the synchronization of the
                     // entries in the set.
                     if refresh_deletes {
-                        delete_uuids.extend(syncuuids.into_iter())
+                        let d_uuids = delete_uuids.get_or_insert_with(Vec::default);
```

Comment:
`get_or_insert_default()` is coming, still unstable for now

Reply:
Yeah, I know. I found the docs for that and was sad I can't use it yet.